### ISSUE TITLE
skin.aeon.nox.silvo

### ISSUE THREAD

**Issue by mikesilvo164** [xbmc/repo-skins#202]
### Add-on details:

- **General**
  - Add-on name: Aeon Nox: SiLVO
  - Add-on ID: skin.aeon.nox.silvo
  - Version number: 7.0.0
  - Kodi/repository version: n/a

- **Code location**
  - URL: https://github.com/mikesilvo164/Aeon-Nox-SiLVO
  - Revision/tag: eeac772
  - Branch: master
  - Kodi/repository version: Leia

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-skins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

**Comment by ronie** [comment]
thanx for submitting your skin to the addon repo!

couldn't find many issues to nag about, so well done! :-)

just a few missing textures for you to look into:


Could not find texture 'flags/studios/default-studio.png'
```
View_50_List.xml:  <texture fallback="flags/studios/default-studio.png">$VAR[StudioIconVar]</texture>
View_50_List.xml:  <texture fallback="flags/studios/default-studio.png">$VAR[StudioIconVar]</texture>
View_50_List.xml:  <texture fallback="flags/studios/default-studio.png">$VAR[StudioIconVar]</texture>
View_50_List.xml:  <texture fallback="flags/studios/default-studio.png">$VAR[StudioIconVar]</texture>
View_50_List.xml:  <texture fallback="flags/studios/default-studio.png">$INFO[ListItem.Label,resource://resource.images.studios.white/,.png]</texture>
View_50_List.xml:  <texture fallback="flags/studios/default-studio.png">$INFO[ListItem.Label,resource://resource.images.studios.coloured/,.png]</texture>
View_50_List.xml:  <texture fallback="flags/studios/default-studio.png">$INFO[ListItem.Label,resource://resource.images.studios.white/,.png]</texture>
View_50_List.xml:  <texture fallback="flags/studios/default-studio.png">$INFO[ListItem.Label,resource://resource.images.studios.coloured/,.png]</texture>
View_602_FanartList.xml:  <texture fallback="flags/studios/default-studio.png">$VAR[StudioIconVar]</texture>
```


Could not find texture 'thumbs/cd_shadow.png'
```
View_54_ShowCase.xml:  <bordertexture border="16">thumbs/cd_shadow.png</bordertexture>
```


Could not find texture 'thumbs/cd_shadow2.png'
```
Includes_Home.xml:  <bordertexture border="28">thumbs/cd_shadow2.png</bordertexture>
MusicVisualisation.xml:  <bordertexture border="28">thumbs/cd_shadow2.png</bordertexture>
MusicVisualisation.xml:  <bordertexture border="28">thumbs/cd_shadow2.png</bordertexture>
```

Could not find texture 'Listselect_fo.png'  (this one is just a case sensitive issue)
```
DialogMediaSource.xml:  <texturefocus colordiffuse="$VAR[HighlightBarColor]" border="5">Listselect_fo.png</texturefocus>
```

**Comment by mikesilvo164** [comment]
Textures issues should be fixed with [940f771](https://github.com/mikesilvo164/Aeon-Nox-SiLVO/commit/940f7719ee2cb4497541b59a7ca2179f74622444)

I had a question regarding script.skin.info.service since I use this for movie set info throughout the skin and have been maintaining it in my repository [here](https://github.com/mikesilvo164/script.skin.info.service). I asked phil65 for permission awhile ago and never heard back but I was just curious if it would be possible to add it to Kodi repo or do you think I should just continue to keep it in my personal one?

Thanks @ronie 😃

**Comment by ronie** [comment]
sure, feel free to create a PR to get your updated version of script.skin.info.service in the addon repo.


i've added aeon nox silvo to the leia repo: https://github.com/xbmc/repo-skins/commit/68456a7ce9ed8ae49eb8371af95f7755ffa877bc
thx!
